kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/wires/PropertiesUi.java

```diff
@@ -628,10 +628,8 @@ private int findNextChannelId() {
 
             @Override
             public String getValue(final GwtChannelInfo object) {
-                final String value = object.get(id).toString();
-                return value == null ? "" : value;
+                return object.get(id).toString();
```

Comment:
Exactly. Previously also, it was erroneous. Just add a Null Check If condition on the get to avoid it.